src/server.cpp

```diff
@@ -2326,7 +2322,9 @@ void Server::fillMediaCache()
 				continue;
 			}
 			// Ok, attempt to load the file and add to cache
-			std::string filepath = mediapath + DIR_DELIM + filename;
+			std::string filepath;
+			filepath.append(mediapath).append(DIR_DELIM).append(filename);
```

Comment:
when you concat strings clang-tidy reports you are doing many string copy + append, (one for each + operator), this model permits to prevent those copies